cmd/books/edit/edit.go

```diff
@@ -37,6 +38,29 @@ func (p *Parser) RunCommand(cmd string, args string) error {
 	return nil
 }
 
+// Completer tries to complete a command and its arguments.
+func (p *Parser) Completer(s string) []string {
+	lst := strings.SplitN(s, " ", 2)
+	if len(lst) < 1 {
```

Comment:
This will never happen. See strings.Split. Check instead if len(lst) == 1 && l[0] == "", or better yet, just check to see if s == "" before trying to split it.

I tested this to verify, and pressing tab without typing anything does indeed complete the authors.